dawn/src/dawn/Compiler/DawnCompiler.h

```diff
@@ -15,37 +15,22 @@
 #ifndef DAWN_COMPILER_DAWNCOMPILER_H
 #define DAWN_COMPILER_DAWNCOMPILER_H
 
+#include "dawn/CodeGen/Options.h"
```

Comment:
why di we have more includes here if we only removed code?

Reply:
Taking a look now.

Reply:
Good call!